packages/react/src/components/inner-proxies.ts

```diff
@@ -7,7 +7,7 @@ import { defineCustomElement as defineIonTabButton } from '@ionic/core/component
 import type { JSX as IoniconsJSX } from 'ionicons';
 import { defineCustomElement as defineIonIcon } from 'ionicons/components/ion-icon.js';
 
-import { /*@__PURE__*/ createReactComponent } from './react-component-lib';
+import { createReactComponent } from './react-component-lib';
```

Comment:
Rollup was warning that this was an invalid usage of the PURE comment. As far as I can tell, `createReactComponent` is used by all the components in this file, so adding PURE here doesn't seem to give us anything.